lib/api/controllers/funnelController.js

```diff
@@ -19,7 +19,7 @@ var
   ServiceUnavailableError = require('kuzzle-common-objects').errors.ServiceUnavailableError,
   Request = require('kuzzle-common-objects').Request;
 
-const MIN_TIME_BEFORE_DUMP_PREV_ERROR = 60000;
+var MIN_TIME_BEFORE_DUMP_PREV_ERROR = 60000;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Replace "var" with "let" or "const" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3504)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define this declaration in a local scope or bind explicitly the property to the global object. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3798)